_posts/2023-11-02-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.44
+author: Priya
+date: 2023-11-02
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Date Difference Calculation - Now with System Time and Specific Date Options:
+Calculate date differences more effectively with additional choices. You can choose between system time, specific dates, and even set calculations to the current date-time in UTC. The date difference component is also conveniently displayed in pipelines (Years, Months, Weeks, etc).
+
+#### Insert Current Date and Time - Now with UTC Option:
+Creating a column and inserting the current date and time is easier than ever. You can now choose to insert the current date and time in UTC using the label and insert value rule, giving you more control over your data.
+
+#### Improved Folder Deletion Handling for Dependency Issues:
```

Comment:
#### Folder deletion now gracefully deletes all its items
Earlier folder deletion would leave out those items in it which had some dependencies on other items. Now the deletion is complete and all its contents are removed. No residual items like datasets and folders remain.